fip-0018.md

```diff
@@ -93,9 +93,48 @@ No changes
 ##### Example
 No changes
 
+### Remove Public Address
```

Comment:
Seems a little strange to have this named singularly but it expects and array. Is this a similar standard we've followed elsewhere? Some frameworks (like JQuery) assume most things are arrays. Are we taking a similar approach or has it been thought through as to what will be arrays and what will be single values?

Reply:
This call was added in FIP-4 and it was made to be consistent with add_pub_address